mailpoet/lib/API/MP/v1/API.php

```diff
@@ -88,6 +88,10 @@ public function getSubscribers(array $filter = [], int $limit = 50, int $offset
     return $this->subscribers->getSubscribers($filter, $limit, $offset);
   }
 
+  public function getSubscribersCount(array $filter = []): int {
```

Comment:
was wondering why this method is defined before the last method and not after it?

Reply:
Two previous methods are `getSubscriber()` and `getSubscribers()`. It doesn't make sense to me put the new one at the end.